src/data/code-samples/en/text-spacing/net-maui.md

```diff
@@ -1,5 +1,30 @@
 # Text spacing - .NET MAUI
 
+MAUI offers a few properties to adjust text spacing:
```

Comment:
Let's replace the first sentence with:
In MAUI, there are several properties to adjust text spacing.